### PR TITLE
Added test case for service

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Initial implementation, with no extended test cases.

### CODE REVIEW

File: service.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- hosts: localhost
+  roles:
+    - service/basic-service
```
Comment (alexander-demicev) [inline service.yml]: Can you add login_ocp role here?

File: roles/service/basic-service/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    state: present
+    definition: "{{ lookup('file', 'service-template.yml') }}"
+
+- name: Wait 30 seconds for resource become available
+  k8s_facts:
+    kind: Service
+    api_version: v1
+    namespace: service-example
+    label_selectors: "name=my-service"
+  register: deploy
```
Comment (alexander-demicev) [inline roles/service/basic-service/tasks/main.yml:13]: You would also need this ```register: deploy and (deploy.resources | length > 0)```

**Review by dymurray — APPROVED** [review]

File: roles/service/basic-service/tasks/main.yml (hunk 2 above)
Comment (dymurray) [reply roles/service/basic-service/tasks/main.yml:13]: I think you want:
```
register: deploy
until: deploy.resources | length > 0
```

**Comment by dymurray** [comment]
Approved because the test case looks good, but Alexanders suggestions make sense to me, I would like to see those changes made.

**Pull request merged**